suite/tests/CMakeLists.txt

```diff
@@ -2948,6 +2948,7 @@ else (UNIX)
     append_link_flags(security-win32.except-execution "/dynamicbase:no")
 
     tobuild(security-win32.except-guard security-win32/except-guard.c)
+    tobuild(security-win32.except-int2d security-win32/except-int2d.c)
```

Comment:
I did not test it on 64 bit but it should work the same.
What is a separate "CL" ? Did you mean a separate pull request ?